cfpb/cfgov-refresh
# wellbeing: Move the FWB JS into the routes directory

The FWB JS needs to be able to include and trigger Analytics code. Moving it into the routes directory was the cleanest solution.

## Additions

- FWB JS routes

## Testing

1. Run `gulp build` and ensure the build doesn't error
2. Turn on the `FWB_RELEASE` flag
3. Interact with http://localhost:8000/consumer-tools/financial-well-being/ and http://localhost:8000/consumer-tools/financial-well-being/results/?65

## Todo
- ES6-ify the JS code
- Consider packaging the assets in the FWB app so that it remains self contained
- Move the `o-scale` code to Capital Framework

## Checklist
